### PR TITLE
Major rework.

### PR DESCRIPTION
**Pull request by cwilso**
Tightened up the intro, streamlined radically, moved history to the end prior to moving it to a separate doc.  Began rationalizing against strategy wiki.

### CODE REVIEW

**Comment by cwilso** [comment]
I should have explicitly noted here - this PR is the result of much collaboration between @tantek and myself; it is not solely my work by a long shot.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,41 @@
 *This document is intended to be a stronger vision statement for the [W3C](https://w3.org/).  This is currently exposed as a work item of the [W3C Advisory Board](https://www.w3.org/2002/ab/), on [the AB wiki](https://www.w3.org/wiki/AB/2021_Priorities#Vision). This document is the result of many people's work, notably Chris Wilson, David Singer, Mike Champion, Tantek Çelik, and the rest of the Advisory Board.*
 
+A New Focus: Improving the Web’s Integrity
+
```
[the remark this answers is not in the document]
Comment (cwilso) [reply README.md:4]: I wanted this to be more focused on a specific change in the mission - not just that it's a change, but that we want to change to focus on improving the integrity of the web. (as further described in the points below.)

I'm happy to further wordsmith this, but can we proceed with this change for now and do so separately?

[the remark this answers is not in the document]
Comment (dwsinger) [reply README.md:4]: OK. Then something like "Vision: the Web built on values" or "Vision: Re-basing the Web on values". We're trying to express a compelling vision for the future, and I fear that this title under-sells that. "New" suggests that there are old foci (and maybe they still exist), "focus" isn't as strong a vision. Not a huge deal but I think we can improve

[the remark this answers is not in the document]
Comment (tantek) [reply README.md:4]: Agreed that we can improve with a more compelling title. I think this is a good enough improvement to land and then further iterate. @dwsinger would you be open to opening an issue on this, something like: stronger Vision title / one-liner?

**Review by tantek — APPROVED** [review]
Having worked on this, and reviewed the comments, I think this is good to land as a net improvement and then continue iterating, especially with editorial fixes.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,41 @@
...
+* championing of fundamentals: accessibility, internationalization, security, and privacy
+* consensus-building, mutual respect, and technical coordination across borders, industries, and organizations around the world
+* driving interoperability across vendors and implementations using open test suites
+* fostering collaboration and interoperability with a strong patent policy and open licenses
+* encouraging incubation in new areas & industries with platforms for discussion & collaboration
+* respected relationships with governments & businesses for providing credible advice.
+* continued security and privacy improvements, including revisiting the current Web platform and exploring how to responsibly improve personal privacy, as well as assessing privacy and security impacts of new capabilities, user scenarios, and innovations
+
+This vision of who we are and our shared organizational values must move into strategic conversations about who we serve, how we serve them, and with whom we partner.
+
```
[the remark this answers is not in the document]
Comment (michaelchampion) [reply README.md:32]: I tend to agree with Dave ; there are 9 points that could use some organization to make the document more compelling.  For example I see 3 major categories: Vision for the web itself, Goals for W3C in pursuing that vision, and Principles W3C must adhere to while pursuing the goals.

Here's an attempt to refactor the 9 points into the 3 categories; each category probably needs some explanatory text:

Vision for the web
- The web should work for everyone
- Its fundamental principles : accessibility, internationalization, security, and privacy (openness?  interoperability?)
-
W3C's program to achieve those goals:

- driving interoperability across vendors and implementations using open test suites
- fostering collaboration and interoperability with a strong patent policy and open licenses
- encouraging incubation in new areas & industries with platforms for discussion & collaboration
- building mutually respectful relationships with governments
- revisiting the current Web platform and exploring how to responsibly improve personal privacy, as well as assessing privacy and security impacts of new capabilities, user scenarios, and innovations relationships with governments & businesses for providing credible advice.
- 
Principles while pursuing them:
- putting the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity; a priority of constituencies built on the excellent work from the TAG's Ethical Web Principles and the HTML Design Principles.
- striving for diversity & inclusion of participants from different geographical locations, gender identities, accessibility needs, and more.

File: README.md [hunk 3]
```diff
@@ -29,50 +57,18 @@ Once the W3C and other open communities had proved the Web's potential, business
...
-* by enabling the easy exchange of information around the globe, it has enabled misinformation to flourish and be exploited for political or commercial gain;
-* a lack of diversity in those participating in the W3C has undoubtedly led to [bias in our specifications](https://en.wikipedia.org/wiki/Algorithmic_bias).
+its openness and anonymity enable scams, phishing, and fraud
+it is too easy to gather personal information, unknown to users: business models that support "free" access to web resources mostly depend on collecting detailed personal information on users without their knowledge, performing exploitive surveillance. The wealth of content funded by these models are a public good; the loss of personal privacy and higher barriers to understanding are not
+by accelerating the easy exchange of information around the globe, it has enabled misinformation to flourish and be exploited for political or commercial gain;
+lack of diversity in participation at the W3C has undoubtedly led to [biases in specifications](https://en.wikipedia.org/wiki/Algorithmic_bias).
```
[the remark this answers is not in the document]
Comment (michaelchampion) [reply README.md:67]: Agree it is a bit run-on.  Not sure if it needs bullets or just shorter sentences.

**Review by michaelchampion — CHANGES_REQUESTED** [review]
The only one of my comments (which echoes one of Dave's) that I feel strongly about is organizing the 9 bullet points under "Our Principles:" in "punchier" way.  9 points is too many -- see  https://en.wikipedia.org/wiki/Rule_of_three_(writing) and https://en.wikipedia.org/wiki/The_Magical_Number_Seven,_Plus_or_Minus_Two 

Most of the points are good, although some wordsmithing would help to reduce redundancy, all they really need is to be elaborations of a smaller number of high level points rather than a list of all sorts of stuff.

**Review by michaelchampion — APPROVED** [review]

**Review by dwsinger — APPROVED** [review]

**Pull request merged**